src/status_im/ui/screens/hardwallet/setup/subs.cljs

```diff
@@ -16,6 +16,13 @@
  (fn [db]
    (get-in db [:hardwallet :flow])))
 
+(re-frame/reg-sub
+ :hardwallet-flow-steps
+ (fn [db]
+   (case (get-in db [:hardwallet :flow])
```

Comment:
`if` ?

and these stings looks weird, maybe better to use keywords?

Reply:
These strings are used to display in UI text "Step 1 of 3"